quint/test/effects/base.test.ts

```diff
@@ -375,48 +375,39 @@ describe('unify', () => {
       assert.isTrue(result.isLeft())
     })
 
-    it('returs error when entity names are cyclical', () => {
-      const e1 = parseEffectOrThrow('Read[v1]')
-      const e2 = parseEffectOrThrow('Read[v1, v2]')
-
-      const result = unify(e1, e2)
-
-      result.mapLeft(e =>
-        assert.deepEqual(e, {
-          location: 'Trying to unify Read[v1] and Read[v1, v2]',
-          children: [
-            {
-              location: 'Trying to unify entities [v1] and [v1, v2]',
-              message: "Can't bind v1 to v1, v2: cyclical binding",
-              children: [],
-            },
-          ],
-        })
-      )
-
-      assert.isTrue(result.isLeft())
-    })
-
-    it('returs error when entity names are cyclical in other way', () => {
-      const e1 = parseEffectOrThrow('Temporal[v1, v2]')
-      const e2 = parseEffectOrThrow('Temporal[v1]')
-
-      const result = unify(e1, e2)
-
-      result.mapLeft(e =>
-        assert.deepEqual(e, {
-          location: 'Trying to unify Temporal[v1, v2] and Temporal[v1]',
-          children: [
-            {
-              location: 'Trying to unify entities [v1, v2] and [v1]',
-              message: "Can't bind v1 to v1, v2: cyclical binding",
-              children: [],
-            },
-          ],
-        })
-      )
-
-      assert.isTrue(result.isLeft())
+    it('can unify entities when a single variable in the lhs appears in a union on the rhs', () => {
+      // E.g., given the unification problem
+      //
+      //    v1 =.= v1 ∪ v2
+      //
+      // We should be able to form a valid substitution iff v1 =.= v2, since
+      // this then simplifies to
+      //
+      //   v1 =.= v1
```

Comment:
Should this be
```suggestion
      //   v1 =.= v1 =.= v2
```

Reply:
That is the implication, and here I am showing just the simplified form. But maybe I should make that more explicit in the comment.

Reply:
I thought the `v1 =.= v2` part was the most relevant here, since that's the part that can later result in an unification error